README.md

```diff
@@ -1,18 +1,44 @@
-# Game On! Auth Service 
+# Spring Security GameOn Auth.
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/bfd8ece69df0405eb5165f00fefc087e)](https://www.codacy.com/app/gameontext/gameon-auth?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=gameontext/gameon-auth&amp;utm_campaign=Badge_Grade)
```

Comment:
Is codacy broken?

Reply:
Codacy went on a short vacation when I used the auth-spring-social project as the template for this one, which meant the gradle & readme lost jacoco/codacy. Restored now =)